src/libstd/alloc.rs

```diff
@@ -125,7 +125,8 @@ fn default_alloc_error_hook(layout: Layout) {
 
 #[cfg(not(test))]
 #[doc(hidden)]
-#[lang = "oom"]
+#[cfg_attr(stage0, lang = "oom")]
+#[cfg_attr(not(stage0), alloc_error_handler)]
 #[unstable(feature = "alloc_internals", issue = "0")]
 pub extern fn rust_oom(layout: Layout) -> ! {
```

Comment:
What abi is this function supposed to have? Here it's `"C"` but elsewhere it seems to be `"Rust"`.

Reply:
Good point, I’ve removed `extern` to make it Rust ABI.

Reply:
Shouldn't the abi be checked as part of the signature check?